pulse_actions/handlers/backfilling.py

```diff
@@ -0,0 +1,63 @@
+"""
+This module is for the following use case:
+
+ - Automatic backfilling:
+
+    * If we use pulse_actions to listen to mozilla-inbound finished
+      *test* jobs, we should be able to re-trigger a job a couple of
+      times (similar to what trigger bot does on try) and backfill
+      back to the last known good
+
+    * Notice that trigger bot has conditions on when to stop
+      re-triggering more than needed
+"""
+import logging
+
+from mozci.mozci import backfill_revlist, trigger_range, query_repo_url_from_buildername
+from mozci.sources.buildapi import FAILURE
+from mozci.sources.pushlog import query_revision_info, query_pushid_range
+
+LOG = logging.getLogger()
+
+MAX_REVISIONS = 20
+TIMES = 4
```

Comment:
We only want 1 appearance of each job on the past revisions.
We're only filling in holes.

On the revision that we see the isse we retrigger that job once (for now - ideally 2) so we end up with 2 failed/warning jobs (in the case of a real regression) or 1 failed/warning job + 1 successful re-triggered job (an intermittent orange)